### PR TITLE
Fixes and expands hardware overview page

### PR DESCRIPTION
**Pull request by ysims**
Expands and fixes the hardware overview page.

### CODE REVIEW

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 1]
```diff
@@ -6,9 +6,33 @@ description: Overview and specification of the hardware used by NUbots.
 slug: /projects/hardware/overview
 ---
 
-The platform used is the NUgus, based off the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling. Because of this, the components of the robot were split to allow for the size of the printer bed and the printer volume. The torso was also modified to have open shoulder covers for shoulder servo cable access.
+NUbots uses the NUgus hardware platform, a modification of the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling.
 
-![Coronal image of the NUgus robot](../../../images/front-robot.png)
-![Sagittal image of the NUgus robot](../../../images/side-robot.png)
+![Coronal and sagittal image of the NUgus robot](../../../images/nugus.png 'Coronal and sagittal image of the NUgus robot')
```
Comment (JosephusPaye) [inline src/book/02-projects/03-hardware/01-overview.mdx]: @tayloryoung6396 Any chance we could get a higher resolution transparent version of these images?

Comment (tayloryoung6396) [reply src/book/02-projects/03-hardware/01-overview.mdx]: Probably

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 2]
```diff
@@ -6,9 +6,34 @@ description: Overview and specification of the hardware used by NUbots.
 slug: /projects/hardware/overview
 ---
 
-The platform used is the NUgus, based off the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling. Because of this, the components of the robot were split to allow for the size of the printer bed and the printer volume. The torso was also modified to have open shoulder covers for shoulder servo cable access.
+NUbots uses the NUgus hardware platform, a modification of the [igus® Humanoid Open Platform](http://www.nimbro.net/OP/). The NUgus robots are 3D printed using fused deposition modelling.
```
Comment (tayloryoung6396) [inline src/book/02-projects/03-hardware/01-overview.mdx]: use?

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 3]
```diff
@@ -6,9 +6,34 @@ description: Overview and specification of the hardware used by NUbots.
...
+## igus® Modifications
+
+The following modifications were made to fit our specific requirements:
+
+- The components of the robot were split to to fit the size of the printer bed and volume.
+- The torso was modified to have open shoulder covers for shoulder servo cable access.
+
+## Servos
+
+The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/), with two in each shoulder, one in each elbow, and two in the neck. The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm), with two in each ankle, three in each hip, and one in each knee.
```
Comment (tayloryoung6396) [inline src/book/02-projects/03-hardware/01-overview.mdx]: 4 shoulder servos

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 4]
```diff
@@ -6,9 +6,34 @@ description: Overview and specification of the hardware used by NUbots.
...
+
+## Cameras
+
+The NUgus robot uses two [FLIR Blackfly S](https://flir.app.boxcn.net/s/x0r0bdsjstdkudjd2dwlxoa5fdvj2gpg/file/418603726810) USB3 cameras with a C-Mount lens positioned in their head, producing images at 170FPS. Two custom-built S-Mount to C-Mount lens adaptors are used to attach two [Lensation 180° fisheye S-Mount lenses](https://www.lensation.de/product/BF10M19828S118C/). Two [20" USB3 micro USB to up angle USB A cables](https://www.angledcables.com/up-angle-a2.html) are used to connect the cameras to the robot's computer.
+
+## Computing
+
+The NUgus robot uses the [Intel NUC7i7bnh](https://ark.intel.com/content/www/us/en/ark/products/95065/intel-nuc-kit-nuc7i7bnh.html) NUC with a [Silverstone NT07-115X](https://www.silverstonetek.com/product.php?pid=442&area=en) cooling fan. It has 8GB of RAM, a 256GB SSD, and a [Robotis CM740 Sub Controller](<http://support.robotis.com/en/product/robotis-op2/sub_controller(cm-740).htm>).
```
Comment (tayloryoung6396) [inline src/book/02-projects/03-hardware/01-overview.mdx]: Is it 8GB or RAM?

File: src/book/02-projects/03-hardware/01-overview.mdx (hunk 2 above)
Comment (ysims) [reply src/book/02-projects/03-hardware/01-overview.mdx]: I can't tell which is right, or are both of them? I changed it to "The NUbots team use the NUgus hardware platform...". Does that sound better?

File: src/book/02-projects/03-hardware/01-overview.mdx (hunk 3 above)
Comment (ysims) [reply src/book/02-projects/03-hardware/01-overview.mdx]: That's what I said? :P

File: src/book/02-projects/03-hardware/01-overview.mdx (hunk 4 above)
Comment (Bidski) [reply src/book/02-projects/03-hardware/01-overview.mdx]: https://www.pccasegear.com/products/33137/corsair-cmso8gx4m2a2133c15-8gb-2x4gb-ddr4-sodimm

Comment (Bidski) [reply src/book/02-projects/03-hardware/01-overview.mdx]: The cooling fan also required custom modifications. The supporting arms were cut off and extra screw holes were drilled to line up with the mounting points on the NUC. A copper spacing plate was also milled to bridge the gap between the NUC processor and the fan.

Comment (tayloryoung6396) [reply src/book/02-projects/03-hardware/01-overview.mdx]: Custom fan to power cable, custom pcb for CM740 -> NUC power, Custom USB NUC to -> CM740 cable

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 5]
```diff
@@ -6,9 +6,33 @@ description: Overview and specification of the hardware used by NUbots.
...
+
+## Networking
+
+The NUgus robot has an Ethernet port as well as WiFi capability. It has two custom-ordered 15cm MHF.4 to RP-SMA wireless antenna cables from the [RFShop](https://rfshop.com.au/), with two RP-SMA antennas which are around 1" long.
+
+## Power
+
+The NUgus robot uses a 3850mAh 4s LiPo battery when not plugged into power. It uses a rebuilt Darwin power board and a custom 16A cylindrical fuse holder. The NUC power board which distributes power to the cooling fan and the NUC is also custom.
```
Comment (Bidski) [inline src/book/02-projects/03-hardware/01-overview.mdx]: The cable providing power to the fan and the NUC is also custom, as is the cable providing sense and speed control to the fan.

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 6]
```diff
@@ -6,9 +6,33 @@ description: Overview and specification of the hardware used by NUbots.
...
+## igus® Modifications
+
+The following modifications were made to fit our specific requirements:
+
+- The components of the robot were split to to fit the size of the printer bed and volume.
+- The torso was modified to have open shoulder covers for shoulder servo cable access.
+
+## Servos
+
+The arms and head of the NUgus robot use [MX64AR servos from Dynamixel](http://www.robotis.us/dynamixel-mx-64ar/), with two in each shoulder, one in each elbow, and two in the neck. The legs use [MX106 servos from Dynamixel](http://support.robotis.com/en/product/actuator/dynamixel/mx_series/mx-106.htm), with two in each ankle, three in each hip, and one in each knee.
```
Comment (Bidski) [inline src/book/02-projects/03-hardware/01-overview.mdx]: Flywheel mounting brackets are custom made (milled from aluminium I think). Same as the bracket on the neck.

Cable clamps on the backs of the knees and on the ankle and hip blocks are 3D printed.

Comment (tayloryoung6396) [reply src/book/02-projects/03-hardware/01-overview.mdx]: servo to servo mounting brackets

Comment (Bidski) [reply src/book/02-projects/03-hardware/01-overview.mdx]: @tayloryoung6396 have you got CAD files for these parts?

File: src/book/02-projects/03-hardware/01-overview.mdx (hunk 4 above)
Comment (Bidski) [reply src/book/02-projects/03-hardware/01-overview.mdx]: USB and fan sense connector are Molex 53398-0471 (1.25 mm pitch PicoBlade header, surface mount, vertical, lead-free, 4 circuits).
Power connectors (on the NUC and fan) are Molex 50-37-5043 (2.50mm Pitch, Mini-SPOX Receptacle Crimp Housing, Single Row, Friction Lock, 4 Circuits, Natural) (this is the [standard 4-pin dynamixel connector](http://support.robotis.com/en/product/actuator/dynamixel/dxl_connector.htm)

CM740->NUC power is a [Yeonho SMH250-02](http://www.yeonho.com/cn/product/product_view.asp?part=device&page=&idx=128&listview=list2) (this is a impossibly hard to get part)

Comment (ysims) [reply src/book/02-projects/03-hardware/01-overview.mdx]: These ones for the connectors? https://au.element14.com/molex/53398-0471/connector-header-4pos-1row-1-25mm/dp/2426363
https://au.element14.com/molex/50-37-5043/connector-rcpt-4pos-1row-2-5mm/dp/9979565

Comment (tayloryoung6396) [reply src/book/02-projects/03-hardware/01-overview.mdx]: The second one looks correct, I don't think the first one is

Comment (Bidski) [reply src/book/02-projects/03-hardware/01-overview.mdx]: Should be this one
https://au.element14.com/molex/51021-0400/connector-housing-rcpt-4pos/dp/9733116?MER=sy-me-pd-mi-acce

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 7]
```diff
@@ -6,6 +6,40 @@ description: Overview and specification of the hardware used by NUbots.
...
+
+There are custom made mounting brackets milled from aluminium for the flywheels and neck. There are 3D printed cable clamps on the back of the knees, angles and hip blocks. The NUgus robot uses servo-to-servo mounting brackets.
```
Comment (Bidski) [inline src/book/02-projects/03-hardware/01-overview.mdx]: `ankle` not `angles`

Comment (Bidski) [reply src/book/02-projects/03-hardware/01-overview.mdx]: Custom servo-to-servo mounting brackets

File: src/book/02-projects/03-hardware/01-overview.mdx [hunk 8]
```diff
@@ -6,6 +6,40 @@ description: Overview and specification of the hardware used by NUbots.
...
+
+## Cameras
+
+The NUgus robot uses two [FLIR Blackfly S](https://www.flir.com/products/blackfly-s-usb3?model=BFS-U3-13Y3C-C) USB3 cameras with a C-Mount lens positioned in their head, producing images at 170FPS. Two custom-built S-Mount to C-Mount lens adaptors are used to attach two [Lensation 180° fisheye S-Mount lenses](https://www.lensation.de/product/BF10M19828S118C/). Two [20" USB3 micro USB to up angle USB A cables](https://www.angledcables.com/parts/rr-aums-xxgr.html#RR-AUMS-20GR) are used to connect the cameras to the robot's computer.
+
+## Computing
+
+The NUgus robot uses the [Intel NUC7i7bnh](https://www.pccasegear.com/products/39044/intel-boxnuc7i7bnh-core-i7-7th-gen-nuc-barebone-kit) with a [Silverstone NT07-115X](https://www.silverstonetek.com/product.php?pid=442&area=en) cooling fan. The cooling fan's supporting arms were removed and extra skrew holes drilled to line up with the mounting points on the NUC. A copper spacing plate was milled to bridge the gap between the NUC processor and the fan.
```
Comment (Bidski) [inline src/book/02-projects/03-hardware/01-overview.mdx]: screw*

Comment (TrentHouliston) [inline src/book/02-projects/03-hardware/01-overview.mdx]: Should we be giving links to product pages like pccasegear, or should we be giving links to the manufacturer's pages? (e.g. https://www.intel.com.au/content/www/au/en/products/boards-kits/nuc/kits/nuc7i7bnh.html)

**Comment by tayloryoung6396** [comment]
I mean you could expand upon the modifications but.. icbf

**Pull request merged**